src/SprykerEco/Zed/Computop/Business/Order/ComputopBusinessOrderFactory.php

```diff
@@ -86,4 +87,12 @@ public function createInitEasyCreditMapper()
     {
         return new EasyCreditMapper();
     }
+
+    /**
+     * @return \SprykerEco\Zed\Computop\Business\Order\Mapper\PostPlace\PayuCeeSingleMapper
+     */
+    public function createInitPayuCeeSingleMapper(): PayuCeeSingleMapper
```

Comment:
1. Why does your create**InitPayuCeeSingleMapper**  returns PayuCeeSingleMapper?
2. Why aren't you returning the interface?

Reply:
Done